wurst/closures/ClosureForGroups.wurst

```diff
@@ -94,7 +94,10 @@ public function forUnitsInRectCounted(rect r, int count, ForGroupCallback c)
 /** Executes the given closure for every unit in range of the given position */
 public function forUnitsInRange(vec2 pos, real radius, ForGroupCallback c)
```

Comment:
This should probably be a new function with a different name to not break existing code (I currently rely on the fact that this does not do the collision size correction automatically and call it with radius + MAX_COLLISION_SIZE - this would effectively cause a lot more units to be grouped and then would be checked twice - once by this code once by mine)